src/libraries/System.ServiceProcess.ServiceController/tests/ServiceBaseTests.cs

```diff
@@ -141,6 +142,11 @@ public void TestOnPauseAndContinueThenStop()
         [ConditionalFact(nameof(IsProcessElevated))]
         public void TestOnExecuteCustomCommand()
         {
+            if (PlatformDetection.IsWindowsServerCore)
+            {
+                throw new SkipTestException("Skip on Windows Server Core"); // https://github.com/dotnet/runtime/issues/43207
+            }
```

Comment:
We can't do this in the ConditionalFact?

Reply:
ConditionalXXX attribute a[ccepts `Type, Member*` or `Member](https://github.com/dotnet/arcade/blob/7e0f9bc02f83d0a561d9edb963d856334f048f0d/src/Microsoft.DotNet.XUnitExtensions/src/Discoverers/ConditionalTestDiscoverer.cs#L158)` so I would have to create a property like IsElevatedAndWindowsServerCore which does not seem worth it.